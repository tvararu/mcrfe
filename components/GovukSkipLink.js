export default () => (
  <a href="#main-content" className="govuk-skip-link">
    Skip to main content
  </a>
);
