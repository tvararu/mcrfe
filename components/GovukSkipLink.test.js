import renderer from "react-test-renderer";
import GovukSkipLink from "./GovukSkipLink";

describe("GovukSkipLink", () => {
  it("renders correctly", () => {
    const component = renderer.create(<GovukSkipLink />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
