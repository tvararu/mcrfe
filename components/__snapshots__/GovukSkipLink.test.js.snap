// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GovukSkipLink renders correctly 1`] = `
<a
  className="govuk-skip-link"
  href="#main-content"
>
  Skip to main content
</a>
`;
